testsuite/tests/typing-extensions/extensions.ocaml.reference

```diff
@@ -60,7 +60,8 @@ val is_s : foo -> bool = <fun>
 Line 2, characters 9-17:
 2 | let a2 = M_S.A 20 (* ERROR: Cannot create a value using a private constructor *)
              ^^^^^^^^
-Error: Cannot create values of the private type foo
+Error: Cannot use private constructor A to create values of the extensible variant type 
+       foo
```

Comment:
This seems like a good opportunity to switch this test file to expect tests!

Reply:
Done in the last commit.

Reply:
I don't think you converted this particular file no.
Would you mind doing it as well?

Reply:
Oh sorry about that, I'm on it.

Reply:
This is now done.